src/style/Provider.tsx

```diff
@@ -0,0 +1,83 @@
+/** @jsx jsx */
+import React, { Fragment } from 'react'
+import { ScreenClassProvider, setConfiguration } from 'react-grid-system'
+import { css, Global, jsx, ThemeProvider, useTheme } from './emotion'
+import { ThemeProps } from './theme'
+import { NeedsChildrenProps } from './util'
+
+const Children: React.FC = ({ children }) => {
+  const theme = useTheme()
+  setConfiguration(theme.grid.rules)
+  return (
+    <Fragment>
+      <Global styles={theme.global.css(theme)} />
+      <Global styles={theme.font.text.css(theme)} />
```

Comment:
We don't need all the global sections for fonts (i.e. text / h1 - h6).  Instead, I think we can stick with `theme.global.css` as sufficient.  Or we can use the above global.css Basically, I don't think giving font any special structure is helpful.